appveyor.yml

```diff
@@ -0,0 +1,57 @@
+environment:
```

Comment:
Appveyor should be actiated now. Could be that we need to reopen this PR for the builds to be started or maybe a push is enough.